qemu/deps/win_driver_install/win_driver_install.py

```diff
@@ -202,7 +202,7 @@ def show_log_output(result_file):
     result_file = r"C:\driver_install.log"
     logger = logging.getLogger("driver_install")
     logger.setLevel(logging.DEBUG)
-    fh = logging.FileHandler(result_file)
+    fh = logging.FileHandler(result_file, mode="a+")

```

Comment:
Hi coli, 

is it ok, one case one debuglog file? as all the cases use the same guest, the log will be long at last